### ISSUE TITLE
Switch default TDVP solver to "exponentiate" ?

### ISSUE THREAD

**Issue by emstoudenmire** [ITensor/ITensorTDVP.jl#50]
Currently the default TDVP solver is "applyexp" which performs well but is not as sophisticated as "exponentiate" (which calls KrylovKit). Originally I had some misunderstandings about the time step parameters expected by exponentiate as compared to applyexp, but after resolving these differences they should perform roughly similarly.

**Comment by mtfishman** [comment]
Yes, I would definitely prefer to just use `KrylovKit.exponentiate`, and put our efforts into fixing any issues we come across with `KrylovKit.exponentiate` instead of supporting our own (less sophisticated) version of Krylov exponentiation.
